doc/lua_api.txt

```diff
@@ -6831,8 +6831,14 @@ Used by `minetest.add_particle`.
         -- `physical = true,` and `collide_with_objects = true,`.
         -- Requires collisiondetection = true to have any effect.
 
-        vertical = false,
-        -- If true faces player using y axis only
```

Comment:
It should probably be documented that this feature existed in the past and has been removed. Else some people might wonder when they see it still used in old mods.

Reply:
Please press ![grafik](https://user-images.githubusercontent.com/7613443/56968533-9bb86c80-6b63-11e9-845e-92ebd7744ecc.png).
:arrow_down: